Assets/SEE/UI/Window/VariablesWindow/VariablesWindowItem.cs

```diff
@@ -0,0 +1,385 @@
+using UnityEngine;
+using SEE.Utils;
+using DG.Tweening;
+using System.Collections.Generic;
+using static RootMotion.FinalIK.RagdollUtility;
+using TMPro;
+using SEE.GO;
+using Crosstales;
+using Michsky.UI.ModernUIPack;
+using System.Linq;
+using System;
+using Microsoft.VisualStudio.Shared.VSCodeDebugProtocol.Messages;
+using UnityEngine.EventSystems;
+using System.Collections;
+
+namespace SEE.UI.Window.VariablesWindow
+{
+    /// <summary>
+    /// Represents an item in the <see cref="VariablesWindow"/>.
+    /// 
+    /// <para>
+    ///     Can be used to group items and/or the represent an actual variable.
+    /// </para>
+    /// </summary>
+    public class VariablesWindowItem : PlatformDependentComponent
+    {
+        /// <summary>
+        /// Path to the prefab for this item.
+        /// </summary>
+        private const string variablesWindowItemPrefab = "Prefabs/UI/VariablesWindow/VariablesWindowItem";
+
+        /// <summary>
+        /// Color for variables.
+        /// </summary>
+        private static readonly Color variableColor = Color.blue.Darker();
+
+        /// <summary>
+        /// The shift per indentation level.
+        /// </summary>
+        private const int indentShift = 22;
+
+        /// <summary>
+        /// The name.
```

Comment:
Documentation should be a bit more descriptive, e.g., "The name of the variable", so it's clear what is referred to.